app/components/modal/order/order.modal.controller.js

```diff
@@ -0,0 +1,105 @@
+(function () {
+    'use strict';
+    angular.module('FoodOrderingApp')
+        .controller('OrderModalController', OrderModalController);
+    OrderModalController.$inject = ['$uibModalInstance', '$sessionStorage', '$state', '$uibModal', '$log', '$rootScope', '$location', 'OrderService'];
+
+    function OrderModalController($uibModalInstance, $sessionStorage, $state, $uibModal, $log, $rootScope, $location, OrderService) {
+
+        var vm = this;
+        vm.order = OrderService.getOrder();
+        vm.quantity = [];
+
+        //Checking if the quantity of the order
+        vm.checkOrder = function (order, quantity) {
+            console.log(order, quantity);
```

Comment:
you have injected $log so, use $log instead of console, and do not log unnecessary, what is the use of console log of quantity of the order here ??. You have created a function for just console the log ??